share/revng/test/tests/abi/build_abi_test_binaries_with_gcc.sh

```diff
@@ -31,12 +31,20 @@ mkdir -p "${OUTPUT_DIRECTORY}"
   -o "${OUTPUT_DIRECTORY}/functions"
 
 # Build the binary for runner to `mmap`
+"${TRIPLE}gcc" -c \
+  ${CFLAGS} -O3 -static \
+  -fno-zero-initialized-in-bss \
+  -ffreestanding \
```

Comment:
Why?

Reply:
For consistency with the other versions where we try to strip binaries. As far as gcc documentation is concerned, adding this flags means that the compiler is not going to use anything but a couple of basic functions like `memset`, `memcpy` and friends.